AsyncDisplayKit/ASControlNode+Subclasses.h

```diff
@@ -20,6 +20,12 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ASControlNode (Subclassing)
 
 /**
+ @abstract Indicates whether or not at least one target was added to the receiver
+ @discussion YES if the receiver has at least one target; NO otherwise.
+ */
+@property (nonatomic, readonly, assign, getter=isTargetAdded) BOOL targetAdded;
```

Comment:
@maicki is this needed?  We should probably declare it internal to the .m file, or just use the instance variable, unless there is a clear usecase for another class (incl. subclass) to use it.  I think we should expect the base class to internally manage this as we should expect any usage or subclass to call addTarget(...).